geth/geth-entrypoint

```diff
@@ -46,6 +46,10 @@ if [ "${OP_NODE_OVERRIDE_CANYON+x}" = x ]; then
     ADDITIONAL_ARGS="$ADDITIONAL_ARGS --override.canyon=$OP_NODE_OVERRIDE_CANYON"
 fi
 
+if [ "${OVERRIDE_FJORD+x}" = x ]; then
+    ADDITIONAL_ARGS="$ADDITIONAL_ARGS --override.fjord=$OVERRIDE_FJORD"
+fi
```

Comment:
I think, this shouldn't be optional. We should exit when the `OVERRIDE_FJORD` env variable isn't supplied.

The check can be like:
```
if [[ -z "OVERRIDE_FJORD" ]]; then
    echo "expected OVERRIDE_FJORD to be set" 1>&2
    exit 2
fi
```
and the start command:
```
# Start service
./geth \
    --datadir="$GETH_DATA_DIR" \
    --verbosity="$VERBOSITY" \
    --http \
    --http.corsdomain="*" \
    --http.vhosts="*" \
    --http.addr=0.0.0.0 \
    --http.port="$RPC_PORT" \
    --http.api=web3,debug,eth,net,engine \
    --authrpc.addr=0.0.0.0 \
    --authrpc.port="$AUTHRPC_PORT" \
    --authrpc.vhosts="*" \
    --authrpc.jwtsecret="$OP_NODE_L2_ENGINE_AUTH" \
    --ws \
    --ws.addr=0.0.0.0 \
    --ws.port="$WS_PORT" \
    --ws.origins="*" \
    --ws.api=debug,eth,net,engine \
    --metrics \
    --metrics.addr=0.0.0.0 \
    --metrics.port="$METRICS_PORT" \
    --syncmode="$OP_GETH_SYNCMODE" \
    --gcmode="$OP_GETH_GCMODE" \
    --maxpeers=100 \
    --rollup.sequencerhttp="$SEQUENCER_HTTP" \
    --rollup.halt=major \
    # --op-network="$OP_NODE_NETWORK" \
    --port="$P2P_PORT" \
    --rollup.disabletxpoolgossip=true \
    --override.fjord=$OVERRIDE_FJORD \
    $ADDITIONAL_ARGS # intentionally unquoted
```